packages/manager/modules/pci/src/projects/project/serving/translations/Messages_en_GB.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
No need for English translations, they will be overwritten

Reply:
I did not understand, I should not provide english translation ?